conf.py

```diff
@@ -41,7 +41,7 @@
 
 # General information about the project.
 project = u'gcc'
-copyright = u'2021, gcc peeps'
+copyright = u'2021, GCC'
```

Comment:
Copyright is the Free Software Foundation

Reply:
Done.